### PR TITLE
Feature/converter

### PR DESCRIPTION
**Pull request by bedbugs-bit**
# Description
Implemented Unit Testing using JUnit5 

## Type of change

Test Numeral converter feature that could lead to unexpected functionality

Test Configuration: JUnit5
Results: Passed test case

- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New unit tests pass locally with my changes

### CODE REVIEW

**Comment by bedbugs-bit** [comment]
It will be effective/helpful to test edge cases for the class where 'I' can be placed before 'V' and 'X', and 'X' can be placed before 'L' and 'C'.

File: src/test/java/org/example/numerals/RomanNumsTest.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import org.junit.jupiter.api.DisplayName;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RomanNumsTest {
+
+
+    @DisplayName("Convert the Roman Numerals to Numbers")
+    @org.junit.jupiter.api.Test
```
Comment (Naboth-tieng) [inline src/test/java/org/example/numerals/RomanNumsTest.java:11]: could be better if they were separated to understand the three configurations were inspected, these are:
I can be placed before V (5) and X (10) to make 4 and 9. 
X can be placed before L (50) and C (100) to make 40 and 90. 
C can be placed before D (500) and M (1000) to make 400 and 900.

File: src/test/java/org/example/numerals/RomanNumsTest.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    void romanToInt() {
+        assertAll(()-> assertEquals(1994, RomanNums.romanToInt("MCMXCIV")),
+                ()->assertEquals(55, RomanNums.romanToInt("LV")),
+                ()->assertEquals(14, RomanNums.romanToInt("XIV")),
+                ()-> assertEquals(1990, RomanNums.romanToInt("MCMXC"))    );
+
```
Comment (Naboth-tieng) [inline src/test/java/org/example/numerals/RomanNumsTest.java:17]: And comments would help other devs understand what exactly you are testing

**Review by Naboth-tieng — COMMENTED** [review]
great test, slight modifications required
